.github/workflows/linux.yml

```diff
@@ -48,15 +76,15 @@ jobs:
           git fetch origin $GITHUB_SHA:temporary-ci-branch
           git checkout $GITHUB_SHA || (git fetch && git checkout $GITHUB_SHA)
       - name: Validate print-config
-        run: sh validate.sh -j 2 -w ghc-9.0.1 -v --lib-only -s print-config
+        run: sh validate.sh -j 2 -w ghc-9.2.1 -v --lib-only -s print-config
```

Comment:
I'm confused: the header for this section says "validate-9_0_1", but we now have `-w ghc-9.2.1`. Also, isn't this duplicated vs section "validate-9_2_1"?